ElementX/Sources/Screens/Timeline/View/ItemMenu/TimelineItemMenuAction.swift

```diff
@@ -8,26 +8,37 @@
 import SFSafeSymbols
 import SwiftUI
 
+@MainActor
 struct TimelineItemMenuActions {
     let reactions: [TimelineItemMenuReaction]
     let actions: [TimelineItemMenuAction]
     let debugActions: [TimelineItemMenuAction]
     
-    init?(isReactable: Bool, actions: [TimelineItemMenuAction], debugActions: [TimelineItemMenuAction]) {
+    init?(isReactable: Bool,
+          actions: [TimelineItemMenuAction],
+          debugActions: [TimelineItemMenuAction],
+          emojiProvider: EmojiProviderProtocol) {
         if !isReactable, actions.isEmpty, debugActions.isEmpty {
             return nil
         }
         
         self.actions = actions
         self.debugActions = debugActions
+        
+        var frequentlyUsed = emojiProvider.frequentlyUsedSystemEmojis().prefix(5).map { TimelineItemMenuReaction(key: $0, symbol: .heart) }
```

Comment:
I don't *think* you need this prefix when there's another one further down 🤔
```suggestion
        var frequentlyUsed = emojiProvider.frequentlyUsedSystemEmojis().map { TimelineItemMenuReaction(key: $0, symbol: .heart) }
```